### PR TITLE
(PUP-3632) Update RPM packaging spec for AIO changes

### PR DESCRIPTION
**Pull request by whopper**
This commit updates the RPM packaging specfile for pathing
changes brought in by the all-in-one agent switchover.

### CODE REVIEW

File: ext/redhat/puppet.spec.erb [hunk 1]
```diff
@@ -336,8 +338,8 @@ fi
 if [ "$1" -ge 1 ]; then
   # The pidfile changed from 0.25.x to 2.6.x, handle upgrades without leaving
   # the old process running.
-  oldpid="%{_localstatedir}/run/puppet/puppetmasterd.pid"
-  newpid="%{_localstatedir}/run/puppet/master.pid"
+  oldpid="%{_localstatedir}/run/puppetlabs/puppetmasterd.pid"
```
Comment (kylog) [inline ext/redhat/puppet.spec.erb]: The oldpid is probably in the same old place?

And do we even need this bit anymore? /cc @haus

**Comment by haus** [comment]
Nothing stands out as crazy. This should work.

File: ext/redhat/puppet.spec.erb (hunk 1 above)
Comment (haus) [inline ext/redhat/puppet.spec.erb]: No, we don't need this anymore.

Comment (haus) [inline ext/redhat/puppet.spec.erb]: I don't know how much cleanup we should be doing here.

**Comment by kylog** [comment]
:+1:

**Pull request merged**